<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReservationsEnCours</class>
 <widget class="QDialog" name="ReservationsEnCours">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1551</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> Hotel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/projetimages/projetimages/e.png</normaloff>:/projetimages/projetimages/e.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView {
    selection-background-color:red;
border-radius:15px;
}
QDateEdit {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(/usr/share/icons/crystalsvg/16x16/actions/1downarrow.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QDialog#ReservationsEnCours
{
		background-image: url(:/projetimages/projetimages/a.jpg);

}
*{
font-family:centry gothlic ;
font-size:15px;
}
#widget
{

	background:rgba(0,0,0,0.7);
border-radius:15px;
}
 QPushButton#pushButton
 {
     background-color: red;

     border-radius:50px;
	  width:100px;

 }

QPushButton
 {
     background-color: red;

     border-radius: 10px;
font-size:22px;


 }

QPushButton#modifierreservation_btn
 {
     background-color: red;

     border-radius: 10px;
font-size:15px;


 }
QPushButton#annulerreservation_btn
 {
     background-color: red;

     border-radius: 10px;
font-size:15px;


 }

QPushButton#terminersejour_btn
 {
     background-color: red;

     border-radius: 10px;
font-size:15px;


 }
QToolButton#toolButton
{
backgroun:red;
border-radius:60px;
}
#label
{
color:white;
background:transparent;

}
#label_2
{
color:white;
background:transparent;

}
#cinresencours{
color:white;
background:transparent;
font:bold;


}
#label_3
{
color:white;
background:transparent;

}
#label_4
{
color:white;
background:transparent;

}
#label_5
{
color:white;
background:transparent;

}
#label_6
{
font-size:25px;

color:white;
background:transparent;

}
#label_7
{
font-size:25px;

color:white;
background:transparent;

}
#Nom
{
color:white;
background:transparent;

}

#nomresencours{
color:white;
background:transparent;
font:bold;


}
#prenomresencours{
color:white;
background:transparent;
font:bold;



}

#label_8
{
color:white;
background:transparent;
font:bold;


}#label_9
{
color:white;
background:transparent;
font:bold;


}#label_10
{
color:white;
background:transparent;
font:bold;


}



QPushButton
{
color:white;

border-raduis:15px;
}



QPushButton#annulerreservation_btn:hover{
color: white;
border-radius:15px;
background:rgb(255, 85, 127);
}


QPushButton#modifierreservation_btn:hover{
color: white;
border-radius:15px;
background:rgb(255, 85, 127);
}


QPushButton#terminersejour_btn:hover{
color: white;
border-radius:15px;
background:rgb(255, 85, 127);
}


QPushButton:hover
{
color: white;
border-radius:15px;
background:rgb(255, 85, 127);
}
QLineEdit#cher_ch_cin1_txt
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1511</width>
     <height>761</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="cher_ch_cin1_txt">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>110</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="typelogement_cb">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>520</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Compteutili_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/projetimages/projetimages/aaa.png</normaloff>:/projetimages/projetimages/aaa.png</iconset>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>1141</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="modifierreservation_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Modifier réservation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="annulerreservation_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>640</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Annuler réservation</string>
    </property>
   </widget>
   <widget class="QComboBox" name="numchambre_cb">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="historique_btn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Historique</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>centry gothlic</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Entrer le numéro de chambre ou le CIN de client à chercher :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="terminersejour_btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>580</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Terminer séjour</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="datearrivee_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="datedepart_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>430</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="client_btn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Clients</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chambreutil_btn">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>40</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Chambres</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouterreservation_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Ajouter réservation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="typelogutil_btn">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Types de logements</string>
    </property>
   </widget>
   <widget class="QLabel" name="Nom">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>centry gothlic</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Type de logement</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date départ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numéro de chambre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date arrivée</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>centry gothlic</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>centry gothlic</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitterutil_btn">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/projetimages/projetimages/aa.png</normaloff>:/projetimages/projetimages/aa.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CIN :</string>
    </property>
   </widget>
   <widget class="QLabel" name="nomresencours">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="prenomresencours">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="cinresencours">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Prénom :</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
